packages/tools/pluggable-widgets-tools/configs/rollup.config.js

```diff
@@ -0,0 +1,278 @@
+                    }
+                }),
+                ...getCommonPlugins({
+                    sourceMaps: !production,
+                    extensions: webExtensions,
+                    transpile: production,
+                    babelConfig: {
+                        presets: [["@babel/preset-env", { targets: { safari: "12" } }]],
+                        allowAllFormats: true
+                    }
+                })
+            ],
+            onwarn
+        });
+    }
+
+    if (platform === "native") {
+        result.push({
+            input: widgetEntry,
+            output: {
+                format: "es",
+                file: join(outDir, outWidgetFile),
+                sourcemap: false
+            },
+            external: nativeExternal,
+            plugins: [
+                ...getClientComponentPlugins(),
+                json(),
+                ...getCommonPlugins({
+                    sourceMaps: false,
+                    extensions: nativeExtensions,
+                    transpile: false
+                })
+            ],
+            onwarn
+        });
+    }
+
+    if (platform === "web" && previewEntry) {
+        result.push({
+            input: previewEntry,
+            output: {
+                format: "commonjs",
+                file: join(outDir, `${widgetName}.editorPreview.js`),
+                sourcemap: !production ? "inline" : false
+            },
+            external: [/^mendix($|\/)/, "react", "react-dom"],
+            plugins: [
+                sass({ output: false, include: /\.(css|sass|scss)$/ }),
+                ...getCommonPlugins({
+                    sourceMaps: !production,
+                    extensions: webExtensions,
+                    transpile: production,
+                    babelConfig: { presets: [["@babel/preset-env", { targets: { safari: "12" } }]] }
+                })
+            ],
+            onwarn
+        });
+    }
+
+    if (editorConfigEntry) {
+        // We target Studio Pro's JS engine that supports only es5 and no source maps
+        result.push({
+            input: editorConfigEntry,
+            output: {
+                format: "commonjs",
+                file: join(outDir, `${widgetName}.editorConfig.js`),
+                sourcemap: false
+            },
+            treeshake: { moduleSideEffects: false },
+            plugins: [
+                ...getCommonPlugins({
+                    sourceMaps: false,
+                    extensions: webExtensions,
+                    transpile: true,
+                    babelConfig: { presets: [["@babel/preset-env", { targets: { ie: "11" } }]] }
+                })
+            ],
+            onwarn
+        });
+    }
+
+    const customConfigPath = join(sourcePath, "rollup.config.js");
+    if (existsSync(customConfigPath)) {
+        const customConfig = await loadConfigFile(customConfigPath, { ...args, configDefaultConfig: result });
+        customConfig.warnings.flush();
```

Comment:
Why are we flushing the warnings 😬

Reply:
~~no idea~~ to show them in console, this is an api of `loadConfigFile` described in [documentation](https://rollupjs.org/guide/en/#programmatically-loading-a-config-file)